src/main/java/com/rocketden/main/service/RoomService.java

```diff
@@ -0,0 +1,69 @@
+package com.rocketden.main.service;
+
+import com.rocketden.main.dao.RoomRepository;
+import com.rocketden.main.dto.room.CreateRoomMapper;
+import com.rocketden.main.dto.room.CreateRoomRequest;
+import com.rocketden.main.dto.room.CreateRoomResponse;
+import com.rocketden.main.dto.room.JoinRoomMapper;
+import com.rocketden.main.dto.room.JoinRoomRequest;
+import com.rocketden.main.dto.room.JoinRoomResponse;
+import com.rocketden.main.model.Room;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Random;
+
+@Service
+public class RoomService {
+
+    public static final int ROOM_ID_LENGTH = 6;
+
+    private final RoomRepository repository;
+
+    @Autowired
+    public RoomService(RoomRepository repository) {
+        this.repository = repository;
+    }
+
+    public JoinRoomResponse joinRoom(JoinRoomRequest request) {
```

Comment:
To be clear, this isn't actually adding anybody to the room? If you wanted, you could add a `List<Player>` (`Player` could be its own object or a `String` -- I'd recommend the `Player` object but if that's too many changes no problem if not) to the `Room` class.

If you didn't add this because you wanted to keep the PR smaller, that's also OK, no need to add that to this PR.

Reply:
I tried adding a list of strings but then got [this error](https://stackoverflow.com/questions/21059451/cannot-declare-list-property-in-the-jpa-entity-class-it-says-basic-attribute), so I think the only way would be to create a new Player entity, which might be beyond the scope of the PR. However I added a card to the backlog for this [here](https://trello.com/c/sj0GCL37/97-actually-add-players-to-rooms).

Reply:
Ok, that sounds OK -- thank you again for adding the card to the Trello board, much appreciated (good name too haha, "**Actually** add players to rooms").